test/local/routes/password.js

```diff
@@ -258,68 +261,126 @@ describe('/password', () => {
     }
   )
 
-  it(
-    '/change/finish',
-    () => {
-      var uid = uuid.v4('binary')
-      var mockDB = mocks.mockDB({
-        email: TEST_EMAIL,
-        uid: uid
-      })
-      var mockPush = mocks.mockPush()
-      var mockMailer = mocks.mockMailer()
-      var mockLog = mocks.spyLog()
-      var mockRequest = mocks.mockRequest({
-        credentials: {
-          uid: uid.toString('hex')
-        },
-        payload: {
-          authPW: crypto.randomBytes(32).toString('hex'),
-          wrapKb: crypto.randomBytes(32).toString('hex'),
-          sessionToken: crypto.randomBytes(32).toString('hex')
-        },
-        query: {
-          keys: 'true'
-        },
-        log: mockLog
-      })
-      var passwordRoutes = makeRoutes({
-        db: mockDB,
-        push: mockPush,
-        mailer: mockMailer,
-        log: mockLog
-      })
+  describe('/change/finish', () => {
+    it(
+      'smoke',
```

Comment:
Yes, it meant a smoke test, since looking through what it was testing, it just tested everything. It's probably cleaner to eventually pull apart into actual unit tests, but, that's what we have at the moment.